packages/select/stories/SelectField.stories.tsx

```diff
@@ -112,3 +116,11 @@ export const SelectFieldStory: Story = ({
 );
 
 SelectFieldStory.storyName = '<SelectField />';
+
+export const SelectFieldProps: Story = () => (
```

Comment:
combine with the selectprops story once they're all in one file. 

If you run into any issues sometimes the use of multiple story names will interfere with the argstable. If that happens we should move away from the tag syntax for the story names and instead use some thing like selectField, _SelectField, etc and let storybook calculate the displayed story name.